api/lib/infrastructure/repositories/organization-learner-repository.js

```diff
@@ -357,21 +357,21 @@ async function updateCertificability(organizationLearner) {
 }
 
 async function countByOrganizationsWhichNeedToComputeCertificability() {
-  const [{ count }] = await knex('organization-learners')
-    .join('organization-features', 'organization-learners.organizationId', '=', 'organization-features.organizationId')
+  const [{ count }] = await knex('view-active-organization-learners')
+    .join('organization-features', 'view-active-organization-learners.organizationId', '=', 'organization-features.organizationId')
     .join('features', 'organization-features.featureId', '=', 'features.id')
     .where('features.key', '=', ORGANIZATION_FEATURE.COMPUTE_ORGANIZATION_LEARNER_CERTIFICABILITY.key)
-    .where('organization-learners.isDisabled', false)
-    .count('organization-learners.id');
+    .where('view-active-organization-learners.isDisabled', false)
```

Comment:
on a plus besoin du isDisabled false si on tape sur la `view-active-organization-learners `non ?

Reply:
La vue ne prend en compte que le deletedAt pour le moment (cf : https://github.com/1024pix/pix/blob/dev/api/db/migrations/20230419155621_add-view-for-not-deleted-learners.js).